packages/components/src/components/link/link.lite.tsx

```diff
@@ -48,9 +48,8 @@ export default function DBLink(props: DBLinkProps) {
 			onClick={(event: ClickEvent<HTMLAnchorElement>) =>
 				state.handleClick(event)
 			}>
-			<Show when={props.text} else={props.children}>
-				{props.text}
-			</Show>
+			{/* prettier-ignore */}
```

Comment:
```suggestion
			{/* prettier-ignore-next-line */}
```

wouldn't this work ?